### PR TITLE
Project Proposal - Build Wallet Core

### PR DESCRIPTION
**Pull request by francisluz**
Hi Team,

Here's our proposal to bring our open-source core library into the OWF repository.

The idea here is to contribute our Crypto Core, which today manages Wallets on Cardano Blockchain, and it will get expanded to Bitcoin and Ethereum.

### CODE REVIEW

File: projects/begin-wallet-core.md
Comment (tkuhrt) [inline projects/begin-wallet-core.md:50]: Can you provide a link to the source code?

File: projects/begin-wallet-core.md
Comment (tkuhrt) [inline projects/begin-wallet-core.md:5]: Can you please let us know how you meet the [growth acceptance criteria defined in the project lifecycle](https://tac.openwallet.foundation/governance/project-lifecycle/#growth-stage)?

**Review by tkuhrt — COMMENTED** [review]
Thank you for the pull request.

File: projects/begin-wallet-core.md
Comment (francisluz) [reply projects/begin-wallet-core.md:5]: Should be here or in the proposal?

Comment (tkuhrt) [reply projects/begin-wallet-core.md:5]: We can add it to the proposal.

File: projects/begin-wallet-core.md
Comment (francisluz) [reply projects/begin-wallet-core.md]: Done

Comment (francisluz) [reply projects/begin-wallet-core.md]: Yes, I read about it, we can squash with DCO once the proposal it approved if that's ok

File: projects/begin-wallet-core.md
Comment (francisluz) [reply projects/begin-wallet-core.md:5]: updated

File: projects/begin-wallet-core.md
Comment (francisluz) [reply projects/begin-wallet-core.md]: Add the strategy in the proposal too

**Comment by francisluz** [comment]
> @francisluz : Will you be able to join the OpenWallet Foundation TAC call on July 24 @ 7:00 AM US/Pacific time to discuss this proposal?

Yes, I'm available to join, please drop me the invite at francisluz@gmail.com

File: projects/begin-wallet-core.md
Comment (skounis) [inline projects/begin-wallet-core.md:57]: All are permissive :+1:

File: projects/begin-wallet-core.md
Comment (skounis) [inline projects/begin-wallet-core.md:6]: What we see in this repository is a library or a fully functional wallet (white labeled)?

File: projects/begin-wallet-core.md
Comment (tkuhrt) [inline projects/begin-wallet-core.md:51]: Looking at the repository, it looks like the last changes were done 4 months ago. What would you say the status of the project is? Do you have a roadmap for additional changes?

File: projects/begin-wallet-core.md
Comment (francisluz) [reply projects/begin-wallet-core.md:6]: For now, it is just the library that can facilitate the creation of a wallet.

File: projects/begin-wallet-core.md
Comment (francisluz) [reply projects/begin-wallet-core.md:51]: Yes, there's some features that can be moved to this library like, support to hardware wallets, and also next on our roadmap is to add support to Bitcoin

**Review by skounis — APPROVED** [review]

**Comment by tkuhrt** [comment]
**Approved at the July 24, 2024 TAC meeting**

@francisluz : Please let us know what you would like to name this lab so that we may create the correct Discord channels and TAC project page.

**Comment by tkuhrt** [comment]
@francisluz : I have created an initial patch for adding this project to the [TAC website](https://tac.openwallet.foundation). We should probably determine the new name prior to merging this proposal and the patch. Let me know your thoughts.

**Comment by francisluz** [comment]
> @francisluz : I have created an [initial patch](https://github.com/openwallet-foundation/tac/compare/main...tkuhrt:tac:begin-wallet-core?expand=1) for adding this project to the [TAC website](https://tac.openwallet.foundation). We should probably determine the new name prior to merging this proposal and the patch. Let me know your thoughts.

Hi @tkuhrt : Let's use the name

Build Wallet Core

**Comment by tkuhrt** [comment]
> Build Wallet Core

@francisluz : sounds good. Would you mind updating the proposal to reflect the name change? Then we can get it merged.

**Comment by francisluz** [comment]
> > Build Wallet Core
> 
> @francisluz : sounds good. Would you mind updating the proposal to reflect the name change? Then we can get it merged.

Done, and what's my DCO Number? so I can amend my commits to fix it

**Comment by tkuhrt** [comment]
> what's my DCO Number? so I can amend my commits to fix it

DCO is the [Developer Certificate of Origin](https://developercertificate.org/). For amending commits, you need to use the `-s` option of git commit. You can find more information on how to do that for previous commits at either of these locations:
- Hyperledger Indy community has [documented steps to fix DCO on previous commits](https://github.com/hyperledger/indy-sdk/blob/main/docs/contributors/signing-commits.md#how-to-sign-previous-commits). 
- [Fix DCO Guide from src-d](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) contains some different steps you can take.

I hope that helps.

**Pull request merged**
